packages/main/src/webComponents/Wizard/Wizard.mdx

```diff
@@ -13,7 +13,114 @@ import { excludePropsForAbstract } from '@sb/utils';
 
 ## Example
 
-<Canvas of={ComponentStories.Default} />
+<Canvas of={ComponentStories.Default} sourceState="none" />
+
+<details>
+
+<summary>Show Code</summary>
+
+```tsx
+const WizardComponent = () => {
+  const [selected, setSelected] = useState('1');
+  const [disabled, setDisabled] = useState<Record<string, boolean>>({ '2': true, '3': true, '4': true });
+  const [hidden, setHidden] = useState<Record<string, boolean>>({ step3Btn: true, finalizeBtn: true });
+  const goToStep2 = () => {
+    setDisabled((prev) => {
+      const { '2': _omit, ...rest } = prev;
+      return rest;
+    });
+    setSelected('2');
+  };
+  const goToStep3 = () => {
+    setDisabled((prev) => {
+      const { '3': _omit, ...rest } = prev;
+      return rest;
+    });
+    setSelected('3');
+  };
```

Comment:
```suggestion
  const [selected, setSelected] = useState(1);

  const goToNextStep = () => {
    setSelected(selected + 1);
  };

  const goToPreviousStep = () => {
    setSelected(selected - 1);
  };
```

Reply:
This approach does not provide the same behavior because you can select all steps from the beginning, which is often undesirable.

Reply:
But you can disable `Next Step` button or bidę it, this is how I am using this, and that's the reason of my suggestion, because when you will have 10 steps, then you need 10 functions `goToXXStep` with this solution, you can have only 2 functions `goNext` and `goBack`, but you need to disable or hide navigation buttons (what you are making)

Reply:
You're right, that would work, but since this is only an example and I like to keep examples simple, I'd like to stick with my approach. Thanks anyway :)